SAMv1.tex

```diff
@@ -658,6 +658,7 @@ \section{Recommended Practice for the SAM Format}
     should be present.
   \item The {\tt NM} tag should be present.
   \end{enumerate}
+\item At present, this specification does not describe how mappings across the ``join'' in a circular reference sequence should be represented.
```

Comment:
Would it be controversial to just say "use supplementary alignments"?

Reply:
Using supplementary alignments (so representing the linear alignment (§1.2) in two separate alignment records?!) is one way to represent it, but other ways exist and (ignoring downstream considerations for the moment) this one doesn't seem like the best way to represent this in SAM.